src/views/model-viewer/ModelViewer.vue

```diff
@@ -131,7 +133,11 @@ export default {
       );
       loading.value = false;
 
-      bimdataViewer.mount("#viewer", initialWindow);
+      if (topicGuid) {
+        bimdataViewer.mount("#viewer"); // The viewer BCFManager plugin is responsible to open the corresponding topic window/layout
```

Comment:
if the topic_guid isn't valid anymore, does it say something or the user lands on an empty viewer?